packages/@react-aria/combobox/src/useComboBox.ts

```diff
@@ -320,7 +320,7 @@ export function useComboBox<T>(props: AriaComboBoxOptions<T>, state: ComboBoxSta
 
   useEffect(() => {
     if (state.isOpen) {
-      return ariaHideOutside([inputRef.current, popoverRef.current]);
+      return ariaHideOutside([inputRef.current, popoverRef.current] as Element[]);
```

Comment:
at this point we should be sure of input and popover ref or we can do 
"   if (state.isOpen && inputRef.current && popoverRef.current) { "